sydent/sydent.py

```diff
@@ -73,6 +73,7 @@ class Sydent:
         'log.path': '',
         'pidfile.path': 'sydent.pid',
         'ed25519.signingkey': '',
+        'bodyTemplate': 'Your code is {token}',
```

Comment:
Should this config key be prefixed with "sms" like the email ones are prefixed with "email"?